modules/medaka/main.nf

```diff
@@ -0,0 +1,47 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName; getProcessName } from './functions'
+
+params.options = [:]
+options        = initOptions(params.options)
+
+process MEDAKA {
+    tag "$meta.id"
+    label 'process_high'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), meta:meta, publish_by_meta:['id']) }
+
+    conda (params.enable_conda ? "bioconda::medaka=1.4.4" : null)
+    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
+        container "https://depot.galaxyproject.org/singularity/medaka:1.4.4--py38h130def0_0"
+    } else {
+        container "quay.io/biocontainers/medaka:1.4.4--py38h130def0_0"
+    }
+
+    input:
+    tuple val(meta), path(reads), path(assembly)
```

Comment:
is this one assembly per sample or is it the same across all samples? In that case I would suggest something like:

```
tuple val(meta), path(reads)
path(assembly)

Reply:
one assembly per sample, so keep as is